graphspace/settings/base.py

```diff
@@ -161,6 +164,6 @@
 }
 
 MAINTENANCE = False
-IS_MAINTENANCE_SCHEDULED = True if os.environ.get('IS_MAINTENANCE_SCHEDULED', "false") == "true" else False
-MAINTENANCE_START_DATETIME = os.environ.get('MAINTENANCE_START_DATETIME', None)
-MAINTENANCE_END_DATETIME = os.environ.get('MAINTENANCE_END_DATETIME', None)
+IS_MAINTENANCE_SCHEDULED = True if os.getenv('IS_MAINTENANCE_SCHEDULED', "false") == "true" else False
+MAINTENANCE_START_DATETIME = os.getenv('MAINTENANCE_START_DATETIME', None)
```

Comment:
@jddanna Can you add some comments here to describe what the format of MAINTENANCE_START_DATETIME/MAINTENANCE_END_DATETIME should look like?

Reply:
Yes, I added comments.